src/preload/index.d.ts

```
@@ -1,8 +1,111 @@
+  Project,
+  ProjectInput,
+  ProjectUpdate,
+  Database,
+  DatabaseInput,
+  DatabaseUpdate,
+  Rule,
+  RuleInput,
+  RuleUpdate,
+  DatabaseSchema
+} from '../main/database/types'
+import type { GenerateRequest, GenerationResult } from '../main/services/data-generator/types'
```

Biome (2.1.2)

[error] Shouldn't redeclare 'GenerationResult'. Consider to delete it or rename it.

'GenerationResult' is defined here:


(lint/suspicious/noRedeclare)